package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@lob/ui-components",
-  "version": "2.0.73",
+  "version": "2.0.74",
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
       "name": "@lob/ui-components",
-      "version": "2.0.73",
+      "version": "2.0.74",
```

Comment:
I _think_ I updated the version correctly where needed

Reply:
I have found `npm version patch --git-tag-version false` to work great

Reply:
Nice, thanks! Much easier 😄